src/types/Input.types.d.ts

```diff
@@ -19,6 +19,7 @@ declare module '@components/atoms/Input' {
     tabIndex?: number;
     ariaLabel?: string;
     ariaLabelledBy?: string;
+    showPasswordToggle?: boolean;
```

Comment:
I think `password` is a special `Input` data type like a `text` input or a `numerical` input. There might be other text we'd want to hide like an API key, or an `email`. A more general variable name like `hideInput` would encompass all of these `Input` data types.

```suggestion
    hideInput?: boolean;
```